packages/e2e-tests/specs/block-grouping.test.js

```diff
@@ -0,0 +1,234 @@
+			// Creating test blocks
+			await insertBlocksOfMultipleTypes();
+
+			// Multiselect via keyboard.
+			await pressKeyWithModifier( 'primary', 'a' );
+			await pressKeyWithModifier( 'primary', 'a' );
+
+			await clickBlockToolbarButton( 'More options' );
+
+			const groupButton = await page.waitForXPath( '//button[text()="Group"]' );
+			await groupButton.click();
+
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
+	} );
+
+	describe( 'Ungrouping', () => {
+		it( 'ungroups an existing group via options menu', async () => {
+			// Create a Group
+			await insertBlocksOfMultipleTypes();
+			await pressKeyWithModifier( 'primary', 'a' );
+			await pressKeyWithModifier( 'primary', 'a' );
+			await transformBlockTo( 'Group' );
+
+			await clickBlockToolbarButton( 'More options' );
+
+			const unGroupButton = await page.waitForXPath( '//button[text()="Ungroup"]' );
+
+			await unGroupButton.click();
+
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
+	} );
+
+	describe( 'Keyboard shortcuts', () => {
+		it( 'groups using keyboard shortcut', async () => {
+			await insertBlocksOfMultipleTypes();
+			await pressKeyWithModifier( 'primary', 'a' );
+			await pressKeyWithModifier( 'primary', 'a' );
+			await transformBlockTo( 'Group' );
+
+			// Group
+			await pressKeyWithModifier( 'primaryAlt', 'g' );
+
+			const allBlocks = await getAllBlocks();
+
+			expect( allBlocks[ 0 ].name ).toBe( 'core/group' );
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
+
+		it( 'ungroups using keyboard shortcut', async () => {
+			// Create a Group
+			await insertBlocksOfMultipleTypes();
+			await pressKeyWithModifier( 'primary', 'a' );
+			await pressKeyWithModifier( 'primary', 'a' );
+			await transformBlockTo( 'Group' );
+
+			// Ungroup (Primary, Shift, Alt)
+			await pressKeyWithModifier( 'secondary', 'g' );
+
+			const allBlocks = await getAllBlocks();
+
+			expect( allBlocks[ 0 ].name ).not.toBe( 'core/group' );
+			expect( await getEditedPostContent() ).toMatchSnapshot();
+		} );
+	} );
+
+	describe( 'Container Block availability', () => {
+		beforeAll( async () => {
+			// Disable the Group block
+			await page.evaluate( () => {
+				const { dispatch } = wp.data;
+				dispatch( 'core/edit-post' ).hideBlockTypes( [ 'core/group' ] );
+			} );
+		} );
+
+		beforeEach( async () => {
+			// Create a Group
+			await insertBlocksOfMultipleTypes();
+			await pressKeyWithModifier( 'primary', 'a' );
+			await pressKeyWithModifier( 'primary', 'a' );
+		} );
+
+		afterAll( async () => {
+			// Re-enable the Group block
+			await page.evaluate( () => {
+				const { dispatch } = wp.data;
+				dispatch( 'core/edit-post' ).showBlockTypes( [ 'core/group' ] );
+			} );
+		} );
+
+		it( 'does not show group transform if container block is disabled', async () => {
+			const availableTransforms = await getAvailableBlockTransforms();
+
+			expect(
+				availableTransforms
+			).not.toEqual( expect.arrayContaining( [
```

Comment:
nitpick: I think `not.toContain( 'Group' );` might be a shorter alternative:
https://jestjs.io/docs/en/expect#tocontainitem

Reply:
Fixed thanks